src/config/AppConfig.ts

```diff
@@ -290,7 +290,7 @@ class AppConfig {
       productFive: new ProductConfig({
         id: env.get('PRODUCT_FIVE').asString(),
         key: PRODUCT_KEY.FIVE,
-        name: this.uniqueIdentifier.generateUniqueString('product-five'),
+        name: this.uniqueIdentifier.generateUniqueString('aaproduct-five'),
```

Comment:
1. Separate the `aa` part from `product-five` with a dash `-`
2. Add a comment on why this is here.